packages/account/src/test-utils/launchNode.ts

```diff
@@ -265,15 +267,19 @@ export const launchNode = async ({
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const [realIp, realPort] = rowWithUrl.split(' ').at(-1)!.trim().split(':'); // e.g. "2024-02-13T12:31:44.445844Z  INFO new{name=fuel-core}: fuel_core::graphql_api::service: 216: Binding GraphQL provider to 127.0.0.1:35039"
 
-        // Resolve with the cleanup method.
-        resolve({
-          cleanup,
-          ip: realIp,
-          port: realPort,
-          url: `http://${realIp}:${realPort}/v1/graphql`,
-          snapshotDir: snapshotDirToUse as string,
-          pid: child.pid as number,
-        });
+        setTimeout(
+          () =>
+            // Resolve with the cleanup method.
+            resolve({
+              cleanup,
+              ip: realIp,
+              port: realPort,
+              url: `http://${realIp}:${realPort}/v1/graphql`,
+              snapshotDir: snapshotDirToUse as string,
+              pid: child.pid as number,
+            }),
+          delay ?? 0
+        );
```

Comment:
Why is this delay needed?

Reply:
I had introduced it when running some previous tests to try and simulate a delayed connection to the node. My new approach will be to run these tests against a devnet node so this is no longer necessary